src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1067,9 +1067,27 @@ private bool CheckPropertyValueKind(SyntaxNode node, BoundExpression expr, BindV
             }
 
             return true;
+
+            bool reportUseSite(MethodSymbol accessor)
+            {
+                UseSiteInfo<AssemblySymbol> useSiteInfo = accessor.GetUseSiteInfo();
+                if (!object.Equals(useSiteInfo.DiagnosticInfo, propertySymbol.GetUseSiteInfo().DiagnosticInfo))
+                {
+                    if (diagnostics.Add(useSiteInfo, propertySyntax))
```

Comment:
>diagnostics.Add(useSiteInfo, propertySyntax) [](start = 24, length = 44)

`return diagnostics.Add(useSiteInfo, propertySyntax);`

Reply:
> ```return diagnostics.Add(useSiteInfo, propertySyntax);```

Adjusted in [https://github.com/dotnet/roslyn/pull/40640](https://github.com/dotnet/roslyn/pull/40640)

---
In reply to: [361552225](https://github.com/dotnet/roslyn/pull/40560#discussion_r361552225) [](ancestors = 361552225)